doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -0,0 +1,305 @@
+# Generates API Docs from the Azure-SDK-for-NET repo
+# Generates an index page for cataloging different versions od the Docs
+
+$ServiceMapping = @{
```

Comment:
This is fine for now but I would like to have a follow-up item that aligns and shares these mappings across the different language repos.

Reply:
Yes, I am not closing the issue just yet. Still have a few things to fix especially as regards management docs.

Reply:
I created an issue to track that https://github.com/Azure/azure-sdk/issues/876